contracts/strategies/pangolin/strategy-png-avax-lost.sol

```diff
@@ -0,0 +1,124 @@
+pragma solidity ^0.6.7;
+
+import "../strategy-png-minichef-farm-base.sol";
+
+/// @notice The strategy contract for Pangolin's AVAX/LOST Liquidity Pool with PNG and LOST rewards
+contract StrategyPngAvaxLost is StrategyPngMiniChefFarmBase {
+    /// @dev LP and Token addresses
+    uint256 public _poolId = 105;
+    address public png_avax_lost_lp = 0x8461681211B49c15e20B3Cfd4c63BE258878B7D9;
+    
+    address public lost = 0x449674B82F05d498E126Dd6615a1057A9c088f2C;
+
+    constructor(
+        address _governance,
+        address _strategist,
+        address _controller,
+        address _timelock
+    )
+        public
+        StrategyPngMiniChefFarmBase(
+            _poolId,
+            png_avax_lost_lp,
+            _governance,
+            _strategist,
+            _controller,
+            _timelock
+        )
+    {}
+
+    /// @notice **** State Mutations ****
+    /// @dev Collect token fees and add liquidity to base pair
+    function harvest() public override onlyBenevolent {
+        IMiniChef(miniChef).harvest(poolId, address(this));
+
+        /// @dev Get balance of native AVAX and wrap AVAX into ERC20 (WAVAX)  
+        uint256 _avax = address(this).balance;              
+        if (_avax > 0) {                                    
+            WAVAX(wavax).deposit{value: _avax}();
+        }
+
+        /// @dev Check token balances, take fee for each token, then update balances
+        uint256 _lost = IERC20(lost).balanceOf(address(this));
+        uint256 _png = IERC20(png).balanceOf(address(this));
+
+        if (_lost > 0) {
+            _swapPangolin(lost, wavax, _lost); 
```

Comment:
I don't see the usual `safeApprove()` lines here, do we not need them anymore?

Reply:
they are already inside the _swapPangolin function (at the strategy base)

Reply:
Copy